keyboards/dz60/rules.mk

```
@@ -21,8 +21,8 @@ CONSOLE_ENABLE = no	# Console for debug(+400)
 COMMAND_ENABLE = no    # Commands for debug and configuration
 SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
 NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
-AUDIO_ENABLE = no
-RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = yes  # Enable per-key backlight LEDs
```

Please use the comments from the [AVR template](https://github.com/qmk/qmk_firmware/blob/1c9e4502181dc36ed9853dec4341765c87530240/quantum/template/avr/rules.mk#L27-L42).

```suggestion
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality 
```